src/graph/entities/node.h

```diff
@@ -11,17 +11,20 @@
 
 typedef int LabelID;
 
+// variable to initialize attributes pointer and avoid read value from NULL
+AttributeSet NULL_ATTRIBUTE_SET;
+
```

Comment:
I think a better place for this variable is `src/graph/entities/attribute_set.h`
more over we would like to initialize this variable to NULL
```c
NULL_ATTRIBUTE_SET = NULL;
```

Reply:
Ok. Done.